L1Trigger/L1TTrackerDTC/python/Producer_Defaults_cfi.py

```diff
@@ -0,0 +1,62 @@
+    DataFormat              = cms.string  ( "Hybrid" ),                                        # hybrid and tmtt format supported
+    OffsetDetIdDSV          = cms.int32   (  1 ),                                              # tk layout det id minus DetSetVec->detId
+    OffsetDetIdTP           = cms.int32   ( -1 )                                               # tk layout det id minus TrackerTopology lower det id
+  ),
+
+  #=== router parameter
+
+  ParamsRouter = cms.PSet (
+    EnableTruncation = cms.bool  ( True  ), # enables emulation of truncation
+    FreqDTC          = cms.double( 360.  ), # Frequency in MHz, has to be integer multiple of FreqLHC
+    TMP_TFP          = cms.int32 (  18   ), # time multiplexed period of track finding processor
+    NumFramesInfra   = cms.int32 (   6   ), # needed gap between events of emp-infrastructure firmware
+    NumRoutingBlocks = cms.int32 (   2   ), # number of systiloic arrays in stub router firmware
+    SizeStack        = cms.int32 (  64   )  # fifo depth in stub router firmware
+  ),
+
+  #=== converter parameter
+
+  ParamsConverter = cms.PSet (
+    WidthRowLUT    = cms.int32 ( 4 ), # number of row bits used in look up table
+    WidthQoverPt   = cms.int32 ( 9 )  # number of bits used for stub qOverPt. lut addr is col + bend = 11 => 1 BRAM -> 18 bits for min and max val -> 9
+  ),
+
+  #=== Tracker parameter
+
+  ParamsTracker           = cms.PSet (
+    NumRegions            = cms.int32 (  9      ), # number of phi slices the outer tracker readout is organized in
+    NumOverlappingRegions = cms.int32 (  2      ), # number of regions a reconstructable particles may cross
+    NumDTCsPerRegion      = cms.int32 ( 24      ), # number of DTC boards used to readout a detector region
+    NumModulesPerDTC      = cms.int32 ( 72      ), # max number of sensor modules connected to one DTC board
+    TMP_FE                = cms.int32 (  8      ), # number of events collected in front-end
+    WidthBend             = cms.int32 (  6      ), # number of bits used for internal stub bend
+    WidthCol              = cms.int32 (  5      ), # number of bits used for internal stub column
+    WidthRow              = cms.int32 ( 11      ), # number of bits used for internal stub row
+    BaseBend              = cms.double(   .25   ), # precision of internal stub bend in pitch units
+    BaseCol               = cms.double(  1.     ), # precision of internal stub column in pitch units
+    BaseRow               = cms.double(   .5    ), # precision of internal stub row in pitch units
+    BendCut               = cms.double(  1.3125 ), # used stub bend uncertainty in pitch units
+    FreqLHC               = cms.double( 40.     )  # LHC bunch crossing rate in MHz
+  ),
+
+  #=== f/w constants
+
+  ParamsFW = cms.PSet (
+    SpeedOfLight = cms.double( 2.99792458       ), # in e8 m/s
+    BField       = cms.double( 3.81120228767395 ), # in T
```

Comment:
this should probably be obtained from the database (the magnetic field is specified as a database record)

Reply:
see comment on Format_Hybrid_cfi.py